quadriga/src/main/java/edu/asu/diging/quadriga/core/service/impl/CollectionManagerImpl.java

```diff
@@ -68,5 +68,21 @@ public Collection editCollection(String id, String name, String description) thr
             throw new CollectionNotFoundException();
         }
     }
+    
+    /**
+     * Deletes a collection from mongodb collection table by _id
```

Comment:
this method is not mongodb specific and shouldn't mention mongo db here. If we switch backend, this method should (in theory) still work the same way. (the repository is mongo db specific).